src/payment/instrument/instrument-response-transformer.ts

```diff
@@ -43,10 +43,11 @@ export default class InstrumentResponseTransformer {
         return vaultedInstruments
             .map(instrument => {
                 switch (instrument.method_type) {
-                    case 'paypal':
-                        return mapToAccountInstrument(instrument);
+                    case 'bank':
```

Comment:
instead of having a switch, you could define your own type guard, this way you are not suppressing the compiler.
https://medium.com/@wittydeveloper/typescript-make-types-real-the-type-guard-functions-814364e8dbe3

```js
map(instrument => isAccountInstrument(instrument) ?
  mapToAccountInstrument(instrument) : 
  mapToCardInstrument(instrument)
);
```